client/grpc-web-cross-browser-transport/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
You can use `extends` (https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) to use a common base `tsconfig.json`, but this might be against the principle of splitting these up.